app/javascript/components/navBar.vue

```diff
@@ -55,7 +55,7 @@
         type="button"
         class="block mr-8 text-gray-500 hover:text-white focus:text-white focus:outline-none"
       >
-        <button class="flex" @click="signOut()">
+        <button class="flex" @click="signOut() :disabled="signOutLoading"">
```

Comment:
```suggestion
        <button class="flex" @click="signOut()" :disabled="signOutLoading">
```
Se te paso una comilla aca